contrib/endpoints/src/api_manager/api_manager_impl.cc

```diff
@@ -95,7 +101,10 @@ utils::Status ApiManagerImpl::Init() {
       }
     }
 
-    return config_loading_status_;
+    if (global_context_->rollout_strategy().compare(
+            std::string(kConfigRolloutStrategy)) != 0) {
+      return config_loading_status_;
```

Comment:
You can define kConfigRolloutStrategy as std::string, so no need to cast for each request.

BTW,  it is better called kConfigRolloutManaged  or kConfigRolloutFixed.